kubernetes/config-maps/nginx-proxy/default.conf.j2

```diff
@@ -5,8 +5,11 @@ map $http_accept $extension {
     application/json json;
 }
 
-{% if nginx_proxy_ssl %}
-server {
+{% if nginx_proxy_allow_http %}
+ server {
+    listen 80       default_server;
+{% else %}
```

Comment:
This condition enables only HTTP when `nginx_proxy_allow_http` is true. What about HTTPS? HTTPS should be enabled in that case too.

Reply:
Done.